packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/categories/CategoriesRestTest.java

```diff
@@ -88,4 +118,18 @@ protected RestCategoryModel createCategoryModelWithIdAndName(final String id, fi
             .name(name)
             .create();
     }
+
+    protected RestCategoryLinkBodyModel createCategoryLinkModelWithId(final String id)
+    {
+        final RestCategoryLinkBodyModel categoryLinkModel = new RestCategoryLinkBodyModel();
+        categoryLinkModel.setCategoryId(id);
+        return categoryLinkModel;
```

Comment:
I thought we had builders for TAS model objects now :)

Reply:
We should :) I wasn't sure if it pays off to create a builder for one field... but you are right - we shouldn't be lazy - I will add the builder 😅